src/neo4j_genai/schema.py

```diff
@@ -0,0 +1,151 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from typing import Any, Dict, List
+
+import neo4j
+from neo4j.exceptions import CypherSyntaxError
+
+
+BASE_ENTITY_LABEL = "__Entity__"
+EXCLUDED_LABELS = ["_Bloom_Perspective_", "_Bloom_Scene_"]
+EXCLUDED_RELS = ["_Bloom_HAS_SCENE_"]
```

Comment:
Not sure if we need these parameters? I copied them from the langchain implementation.

Reply:
I don't know much cypher, but looking at [the queries](https://github.com/neo4j/neo4j-genai-python/blob/eed26a0c4aeee51cc3eddf5f4f3355a5d9aca9af/src/neo4j_genai/schema.py#L24), these labels and relationships won't get returned anyway unless they have properties? So maybe it's redundant anyway (assuming the auto-generated entities don't have properties)? 🤔